johnwayodi/sm-server
# #161695595 Remove Duplicate Code, Reduce Complexity

#### What does this PR do?
Remove duplicate code and instead put them in a reusable function
#### Description of Task to be completed?
1. Remove duplicate checks for identity from all views
2. Ensure tests are passing after rework
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/sm-api-v2.git`
2. Ensure you are in the cloned repo, **sm-api-v2** by default.
3. Create a virtual environment using virtualenv and pip install -r requirements.txt
4. `git checkout -b ch-remove-duplication-161695595 origin/ch-remove-duplication-161695595`
5. Run **_pytest -vv --cov=storemanager/api/v2_** in terminal to run the tests and get coverage on the application.
6. Run the flask application and test using Postman.
#### Any background context you want to provide?
No Background context for this section
#### What are the relevant pivotal tracker stories?
[#161695595](https://www.pivotaltracker.com/story/show/161695595)
#### Questions:
Any recommendations on how to correctly use global variables in class methods?
